src/provider/cloud-element-templates/index.js

```diff
@@ -19,10 +20,12 @@ export default {
   __init__: [
     'elementTemplatesLoader',
     'replaceBehavior',
-    'elementTemplatesPropertiesProvider'
+    'elementTemplatesPropertiesProvider',
+    'conditionChecker'
```

Comment:
Can we rename this to prevent collisions, i.e. to `elementTemplatesConditionChecker`?